starter-code/src/clue.js

```diff
@@ -1,14 +1,200 @@
 // ITERATION 1
 
 // Suspects Collection
-const suspectsArray = [];
+const suspectsArray = [
+    mrGreen = {
+        firstName: "Jacob",
+        lastName: "Green",
+        occupation: "Entrepreneur",
+        age: 45,
+        description: "He has a lot of connections",
+        image: "https://pbs.twimg.com/profile_images/506787499331428352/65jTv2uC.jpeg",
+        color: "green"
+    },
+
+    drOrchid = {
+        firstName: "Doctor",
+        lastName: "Orchid",
+        occupation: "Scientist",
+        age: 26,
+        description: "PhD in plant toxicology. Adopted daughter of Mr. Boddy",
+        image: "http://www.radiotimes.com/uploads/images/Original/111967.jpg",
+        color: "white"
+    },
+
+    profPlum = {
+        firstName: "Victor",
+        lastName: "Plum",
+        occupation: "Designer",
+        age: 22,
+        description: "Billionaire video game designer",
+        image: "https://66.media.tumblr.com/ee7155882178f73b3781603f0908617c/tumblr_phhxc7EhPJ1w5fh03_540.jpg",
+        color: "purple"
+    },
+
+    missScarlet = {
+        firstName: "Kasandra",
+        lastName: "Scarlet",
+        occupation: "Actor",
+        age: 31,
+        description: "She is an A-list movie star with a dark past",
+        image: "https://www.radiotimes.com/uploads/images/Original/111967.jpg",
+        color: "red"
+    },
+
+    mrsPeacock = {
+        firstName: "Eleanor",
+        lastName: "Peacock",
+        occupation: "Socialité",
+        age: 36,
+        description: "She is from a wealthy family and uses her status and money to earn popularity",
+        image: "https://metrouk2.files.wordpress.com/2016/07/mrs-peacock.jpg",
+        color: "blue"
+    },
+
+    mrMustard = {
+        firstName: "Jack",
+        lastName: "Mustard",
+        occupation: "Retired Football player",
+        age: 62,
+        description: "He is a former football player who tries to get by on his former glory",
+        image: "https://static.independent.co.uk/s3fs-public/thumbnails/image/2016/07/04/08/unspecified-3.jpg",
+        color: "yellow"
+    }
+];
```

Comment:
You can't declare or initialize a variable inside an array (here you're initializing mrGreen, mrMustard etc, inside the suspectsArray). In that case it works, because JS is a weakly typed language and thus is quite permissive, but if you try to declare the variables with **var** mrGreen = {} inside the array, you'll see that you get an error. 
What you could do is declare each object outside the array and then push them to the suspectArray, or just add the suspects objects without assigning them an identifier.